lte/gateway/c/oai/tasks/amf/amf_config.c

```diff
@@ -141,7 +152,364 @@ void amf_config_init(amf_config_t* config) {
 ***************************************************************************/
 int amf_config_parse_opt_line(
     int argc, char* argv[],
-    amf_config_t* config_pP) {  // This function will be removed in upcoming PR
-  amf_config_init(config_pP);
+    amf_config_t* config_aA) {  // This function will be removed in upcoming PR
```

Comment:
What is the significance of `aA`?

Reply:
I think the naming is short hand for config profile pointer. So let's use the same name for consistency and proper semantics.